docs/application.rst

```diff
@@ -56,6 +56,9 @@ Warehouse serves three main classes of users:
    Dustin Ingram, and Donald Stufft, who add classifiers, ban
    spam/malware projects, help users with account recovery, and so
    on. There are under ten such admins.
+4. *PyPI application moderators*. These users have a subset of the
+   permissions of *PyPI application administrators* to assist in some
+   routine administration tasks.
```

Comment:
Not sure which commit you want to add this in, but the paragraph above this list needs updated as well ("three main classes of users"). Probably makes sense to move the things moderators can do, like "who add classifiers" to here instead, and be more explicit about what "routine administration tasks" entails.

Reply:
I reordered these, fixed the count before this, and expanded on what moderators do.